test/ui/ViewJobRolesTests.ts

```diff
@@ -1,100 +1,61 @@
-import { ViewJobRolesTestsPage } from './ViewJobRolesTestsPage';
+import { ViewJobRolesTestsPage } from '../ui/ViewJobRolesTestsPage';
+import { HeaderTestPage } from '../ui/HeaderTestPage';
+import { FooterTestPage } from '../ui/FooterTestPage';
 
 describe('View Job Roles Tests', function () {
-    this.timeout(50000);
+    this.timeout(70000);
+    const page:string = 'https://jptw3amsi2.eu-west-1.awsapprunner.com/jobRoles';
 
     before(async function () {
         await ViewJobRolesTestsPage.startDriver();
+        FooterTestPage.driver = ViewJobRolesTestsPage.getDriver(); // the footer test uses the same driver
+        HeaderTestPage.driver = ViewJobRolesTestsPage.getDriver(); // the header test uses the same driver
```

Comment:
header and footer shouldn't need their own driver. instead have the ViewJobRolesTestsPage extend the header and footer and access the methods through ViewJobRolesTestsPage